lib/ask-stack-result-view.coffee

```diff
@@ -198,7 +215,17 @@ class AskStackResultView extends ScrollView
           atom.workspace.activatePreviousPane()
           # editor = atom.workspace.activePaneItem
           editor = atom.workspace.getActivePaneItem()
-          editor.insertText(code)
+          if qa == true
+            editor.insertText("Insert from Stack Overflow", {select: true})
+            editor.toggleLineCommentsInSelection();
+            editor.insertNewlineBelow();
+            editor.insertText("Author: #{author_name} - #{author_src}", {select: true})
+            editor.toggleLineCommentsInSelection();
```

Comment:
I agree, and full disclosure I'm not 100% happy about this either - I spent several evenings trying to find a documented (or even undocumented but available in the source code) way of being able to add in comments, in bulk, all at once and on several lines. This was the only way I found that could achieve that. However, now that you have raised it - I'll ask the question in one of the atom forums.